### PR TITLE
Adding recipes "Goose*"; "mv_regex"; "pdfnofonts"

### PR DESCRIPTION
**Pull request by tdegeus**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by tdegeus** [comment]
@conda-forge/help-python Ready for review

File: recipes/elasticconstants/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools
+  run:
+    - python
+    - setuptools
```
Comment (scopatz) [inline recipes/elasticconstants/meta.yaml]: Does this really need setuptools at runtime?

File: recipes/goosebib/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    - GbibSelectAlias = GooseBib.cli.GbibSelectAlias:main
+
+requirements:
+  host:
+    - python >=3
+    - pip
+    - setuptools
+  run:
+    - python >=3
+    - setuptools
```
Comment (scopatz) [inline recipes/goosebib/meta.yaml]: Same

File: recipes/goosempl/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+requirements:
+  host:
+    - python >=3
+    - pip
+    - setuptools
+  run:
+    - python >=3
+    - setuptools
+    - numpy
+    - matplotlib
```
Comment (scopatz) [inline recipes/goosempl/meta.yaml]: Should this be `matplotlib-base`?

File: recipes/elasticconstants/meta.yaml (hunk 1 above)
Comment (tdegeus) [reply recipes/elasticconstants/meta.yaml]: @scopatz Thanks for you review. I am currently using `pkg_resources` (shipped with setuptools) to extract the version of my library. But I agree that this is a bit convoluted (I did not realise before packaging here that `pkg_resources` is not standard). So what I will do is update the source, and remove the dependency here, which is much cleaner.

File: recipes/goosempl/meta.yaml (hunk 3 above)
Comment (tdegeus) [reply recipes/goosempl/meta.yaml]: @scopatz Looking at https://github.com/conda-forge/matplotlib-feedstock/blob/master/recipe/meta.yaml I guess you are right. Out of curiosity: Why is the package called `matplotlib-base` and not `matplotlib`? Any why isn't the feedstock called `matplotlib-base`?

**Comment by tdegeus** [comment]
@conda-forge/help-python Ready for review

File: recipes/goosempl/meta.yaml (hunk 3 above)
Comment (chrisburr) [reply recipes/goosempl/meta.yaml]: It probably used to only contain `matplotlib` and the split that created `matplotlib-base` was probably added afterwards.

It's then a quirk of the way multiple `outputs` interact with the README is automatically generated.

**Pull request merged**